### PR TITLE
feat: Add IONOS Virtual Data Center

### PR DESCRIPTION
**Pull request by Jelledb**

### CODE REVIEW

**Comment by felixzieger** [comment]
Let's put IONOS on the root level of the repo.

**Review by ishabakeh — CHANGES_REQUESTED** [review]
Looks good, 
Would you change the logo file name to just "icon.png" it will be leveraged in other areas, we have to also change it in other modules.

**Comment by ishabakeh** [comment]
> Let's put IONOS on the root level of the repo.

I am more in favor of "other providers", otherwise we have to move everything to the root of the repo

File: other-providers/ionos/virtual-data-center/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+   - Implementation Type -> Terraform
+   - Terraform Version -> 1.5
+   - Git Repository URL -> This repository, or, if you fork the code, your own repository
+   - Git Repository Path -> `other-providers/ionos/virtual-data-center`
+   - Enter a SSH Key if you are running from a private repository.
+   - Enter a known host (if you are running from a self-hosted Git provider)
+   - Deletion Mode -> pick "Delete Resources" if you want a `tf destroy` to run upon deletion in meshStack.
+10. Go to the next page and skip the Dependency selection
+11. For the Inputs, create the following Inputs:
+    - `IONOS_TOKEN`
```
Comment (malhussan) [inline other-providers/ionos/virtual-data-center/README.md]: t: or IONOS_USERNAME and IONOS_PASSWORD

**Review by malhussan — APPROVED** [review]
IMO it could also fit in top level as a platform that can run workload and can possibly expand to submodules for different offered IONOS services

**Comment by Jelledb** [comment]
> IMO it could also fit in top level as a platform that can run workload and can possibly expand to submodules for different offered IONOS services

This argument in the end made me choose for the top level. I think it makes a lot of sense.

**Review by ishabakeh — APPROVED** [review]

**Pull request merged**